frontend/src/libs/helpers/check-is-selected-route.ts

```diff
@@ -0,0 +1,20 @@
+import { AppRoute } from '#libs/enums/app-route.enum.js';
```

Comment:
we should fully remove this helper and check it another way. I meant that the idea of checking by `startsWith` isn't good, we should strictly compare routes, that's why we need to pass it to config somehow

Reply:
@Liza-Veis , I'm sorry, I forgot to check for a response again. I couldn't find a route config? The only thing I found is SIDEBAR_ROUTES, but that's what we are passing as routes to the Navigation menu. Can you give me a bit details, so I could close it today already?